cli/src/utils/helpers.ts

```diff
@@ -52,3 +53,11 @@ export async function getProjectPackageJson() {
   }
   return loaded;
 }
+
+export function getPrimaryDomain(environment: Environment) {
+  return(`${environment.domains.filter((d) => d.type === 'subdomain')[0].name}`);
```

Comment:
Are we relying on ordering or assuming exactly one subdomain? Could possibly be an additional property on a domain marking at as primary to avoid logic on client.

Reply:
we still assume there's only one subdomain (the future PR does not assume order of cnames)